controller.js

```diff
@@ -52,7 +50,35 @@ module.exports = {
         data: err
       });
     })
-  }
+  },
+
+
+/**
+  * fetches a single active news record from the database
+  * @function fetchSingleActiveNews
+  * @param {String} status: a value either 'Y' or 'N' for active and inactive news
+  * @return {Promise} Promise having a news record with title and Linkimg
+*/
+fetchSingleActiveNews:function(status){
+  return models.News.findOne({
+     where:{
+       is_Active: status
+     }
+   })
+  },
 
+/**
+  * fetches all the active news record from the database
+  * @function getActiveNews
+  * @return {Promise} Promise having the all the active news records
+*/
+  getActiveNews:function(){
+   return models.News.findAll({
+      where:{
+        is_Active:'Y'
```

Comment:
is this string case as required by the DB? Feel "is_Active" is mixed case that we shouldn't use.